pkg/volume/iscsi/iscsi.go

```diff
@@ -97,6 +97,11 @@ func (plugin *iscsiPlugin) newBuilderInternal(spec *volume.Spec, podUID types.UI
 	lun := strconv.Itoa(iscsi.Lun)
 	portal := portalBuilder(iscsi.TargetPortal)
 
+	iface := iscsi.ISCSIInterface
+	if iface == "" {
+		iface = "default"
```

Comment:
Is this something that you should do in API-level defaulting?  The way it is, a user will not see the value 'default' if they get their pod by API.  Should they?  when our API docs says "defaults to" we generally mean that.  Not forcing, just asking what the best user experience will be.